cpp/Platform.Equality.Tests/Platform.Equality.Tests.cpp

```diff
@@ -0,0 +1,130 @@
+#include <Platform.Equality.h>
+#include <gtest/gtest.h>
+
+#include <span>
+#include <list>
+
+namespace Platform::Equality
```

Comment:
The namespace here should be `Platform::Equality::Tests` the same as the folder.